### DISCUSSION TITLE
VS Code Remote integration

### DISCUSSION THREAD

**Discussion by homoky** [cypress-io/cypress#9194] [Questions and Help]
Hello.

We have started using cypress while ago but recently we moved to VS Code Remote Workspaces. `cypress run` command works fine, but opening the application (browser) via SSH terminal does not work (`cypress open`). 

Do you have any experience with this dev setup so you can help me out?

Thank you.

**Comment by bahmutov** [comment]
It is tricky. We have done it via ssh reverse tunnel but it wasn’t pretty

**Comment by hju-mikekevin-castro** [comment]
We too in our company are using VS Code Remote Workspaces. Would this be possible even if we use `cypress run`?

**Comment by bahmutov** [comment]
Yes, cypress run works (provided the target machine uses a container with Cypress dependencies installed, like one of the Cypress Docker images). It is just cypress open that does not show anything that I would love to solve
